x-pack/plugin/eql/qa/common/src/main/java/org/elasticsearch/test/eql/CommonEqlActionTestCase.java

```diff
@@ -195,8 +202,8 @@ else if (hits.sequences() != null) {
         }
     }
 
-    protected EqlSearchResponse runQuery(String index, String query) throws Exception {
-        EqlSearchRequest request = new EqlSearchRequest(testIndexName, query);
+    protected EqlSearchResponse runQuery(String index, String query, boolean isCaseSensitive) throws Exception {
```

Comment:
Maybe add two more methods - `runQueryCaseSensitive` and `runQueryCaseInsensitive` - that should "hide" the boolean parameter choice and call these instead of `runQuery(...., true/false)`?